<?xml version="1.0"?>
<launch>

  <node name="thrusters_controller" pkg="thrusters_controllers"
        type="thrusters_controller_node" output="screen">
    <rosparam command="load" file="$(find thrusters_controllers)/config/rexrov.yaml" />
    <remap to="/rexrov/cmd_vel" from="/twist_command"/>
  </node>

  <include file="$(find uuv_teleop)/launch/uuv_teleop.launch">
    <arg name="uuv_name" value="rexrov" />
    <arg name="joy_id" value="0" />
  </include>

  <node name="sim_transform" pkg="topic_tools" type="transform" args="/thrusters/0 /rexrov/thrusters/0/input std_msgs/Float64 m">
  </node>
</launch>
